### PR TITLE
[1.1.1.1] Added instructions on how to set up on Windows 11

### PR DESCRIPTION
**Pull request by marciocloudflare**

### CODE REVIEW

File: products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md [hunk 1]
```diff
@@ -1,86 +1,87 @@
...
 1. Click the **Start menu** > **Settings**.
-1. Select **Network and Internet** > **Change Adapter Settings**.
-1. Right-click on the WiFi network you are connected to and click **Properties**.
-1. Select **Internet Protocol Version 4**
-1. Click **Properties**.
-1. Click **Use The Following DNS Server Addresses**.
+1. Select **Network and Internet** > **Change Adapter Options**.
+1. Right-click on the Ethernet or WiFi network you are connected to and click **Properties**.
+1. Select **Internet Protocol Version 4**.
+1. Click **Properties** > **Use The Following DNS Server Addresses**.
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md]:
```suggestion
1. Click **Properties** > **Use the following DNS server addresses**.
```
(I checked the options I have available.)

File: products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md [hunk 2]
```diff
@@ -1,86 +1,87 @@
...
-
-7. Click **OK**.
+1. Remove any DNS addresses that may be already listed.
+1. Add `1.1.1.2` to the **Preferred DNS server** field, and `1.0.0.2` to the **Alternate DNS server** field.
+1. Click **OK**.
+1. Select **Internet Protocol Version 6**.
+1. Select **Properties** > **Use The Following DNS Server Addresses**.
+1. Take note of any DNS addresses you might have and save them in a safe place in case you need to use them later.
+1. Remove any DNS addresses that may be already listed.
+1. Add `2606:4700:4700::1112` to the **Preferred DNS** field, and `2606:4700:4700::1002` to the **Alternate DNS** field.
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md]:
```suggestion
1. Add `2606:4700:4700::1112` to the **Preferred DNS server** field, and `2606:4700:4700::1002` to the **Alternate DNS server** field.
```

File: products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md [hunk 3]
```diff
@@ -1,86 +1,87 @@
...
-    1.1.1.2
-    1.0.0.2
-    ```
-
-7. Click **OK**.
+1. Remove any DNS addresses that may be already listed.
+1. Add `1.1.1.2` to the **Preferred DNS server** field, and `1.0.0.2` to the **Alternate DNS server** field.
+1. Click **OK**.
+1. Select **Internet Protocol Version 6**.
+1. Select **Properties** > **Use The Following DNS Server Addresses**.
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md]:
```suggestion
1. Select **Properties** > **Use the following DNS server addresses**.
```

File: products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md [hunk 4]
```diff
@@ -1,86 +1,87 @@
...
+### Block malware and adult content
 
 1. Click the **Start menu** > **Settings**.
-1. Click **Network and Internet** > **Change Adapter Settings**.
-1. Right-click on the Wi-Fi network you are connected to and click **Properties**.
-1. Select **Internet Protocol Version 6**.
+1. Select **Network and Internet** > **Change Adapter Options**.
+1. Right-click on the Ethernet or WiFi network you are connected to and click **Properties**.
+1. Select **Internet Protocol Version 4**.
 1. Click **Properties** > **Use The Following DNS Server Addresses**.
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md]:
```suggestion
1. Click **Properties** > **Use the following DNS server addresses**.
```

File: products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md [hunk 5]
```diff
@@ -1,86 +1,87 @@
...
-    2606:4700:4700::1112
-    2606:4700:4700::1002
-    ```
-
-7.  Click **OK**.
+1. Remove any DNS addresses that may be already listed.
+1. Add `1.1.1.3` to the **Preferred DNS server** field, and `1.0.0.3` to the **Alternate DNS server** field.
+1. Click **OK**.
+1. Select **Internet Protocol Version 6**.
+1. Select **Properties** > **Use The Following DNS Server Addresses**.
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md]:
```suggestion
1. Select **Properties** > **Use the following DNS server addresses**.
```

File: products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md [hunk 6]
```diff
@@ -1,86 +1,87 @@
...
+1. Take note of any DNS addresses you might have and save them in a safe place in case you need to use them later.
+1. Remove any DNS addresses that may be already listed.
+1. Add `2606:4700:4700::1113` to the **Preferred DNS** field, and `2606:4700:4700::1003` to the **Alternate DNS** field.
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md]:
```suggestion
1. Add `2606:4700:4700::1113` to the **Preferred DNS server** field, and `2606:4700:4700::1003` to the **Alternate DNS server** field.
```

File: products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md [hunk 7]
```diff
@@ -1,86 +1,87 @@
...
+1. Click **OK**.
 
-## Block malware and adult content
+## Windows 11
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md:49]: The set of instructions for Windows 11 seems exactly the same as for Windows 10... Am I missing something?

If they're the same, we could keep "Windows" as the heading and add an introductory sentence saying that it applies both to Windows 10 and Windows 11.

File: products/1.1.1.1/src/content/setup-1.1.1.1/windows.md [hunk 8]
```diff
@@ -1,42 +1,47 @@
...
-Follow these steps to configure 1.1.1.1:
+## Windows 10
 
 1. Click the **Start menu** > **Settings**.
-1. Select **Network and Internet** > **Change Adapter Settings**.
-1. Right-click on the WiFi network you are connected to and click **Properties**.
+1. Select **Network and Internet** > **Change Adapter Options**.
+1. Right-click on the Ethernet or WiFi network you are connected to and click **Properties**.
 1. Select **Internet Protocol Version 4**.
 1. Click **Properties** > **Use The Following DNS Server Addresses**.
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/setup-1.1.1.1/windows.md]:
```suggestion
1. Click **Properties** > **Use the following DNS server addresses**.
```

File: products/1.1.1.1/src/content/setup-1.1.1.1/windows.md [hunk 9]
```diff
@@ -1,42 +1,47 @@
...
-    1.1.1.1
-    1.0.0.1
-    ```
-
+1. Remove any DNS addresses that may be already listed.
+1. Add `1.1.1.1` to the **Preferred DNS server** field, and `1.0.0.1` to the **Alternate DNS server** field.
 1. Click **OK**.
-1. Now, go to **Internet Protocol Version 6**.
+1. Select **Internet Protocol Version 6**.
 1. Select **Properties** > **Use The Following DNS Server Addresses**.
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/setup-1.1.1.1/windows.md]:
```suggestion
1. Select **Properties** > **Use the following DNS server addresses**.
```

File: products/1.1.1.1/src/content/setup-1.1.1.1/windows.md [hunk 10]
```diff
@@ -1,42 +1,47 @@
...
 1. Take note of any DNS addresses you might have and save them in a safe place in case you need to use them later.
-1. Remove any DNS addresses that may be already listed and in their place add:
+1. Remove any DNS addresses that may be already listed.
+1. Add `2606:4700:4700::1111` to the **Preferred DNS** field, and `2606:4700:4700::1001` to the **Alternate DNS** field.
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/setup-1.1.1.1/windows.md]:
```suggestion
1. Add `2606:4700:4700::1111` to the **Preferred DNS server** field, and `2606:4700:4700::1001` to the **Alternate DNS server** field.
```

File: products/1.1.1.1/src/content/setup-1.1.1.1/windows.md [hunk 11]
```diff
@@ -1,42 +1,47 @@
...
-    2606:4700:4700::1111
-    2606:4700:4700::1001
-    ```
+## Windows 11
 
-1. Click **Close**.
+1. Click the **Start menu** > **Settings**.
+1. Select **Network and Internet**.
+1. Click the adapter you want to configure - like as your Ethernet adapter or Wi-Fi card.
+1. Scroll to **DNS server assignment** and click **Edit**.
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/setup-1.1.1.1/windows.md:36]: These instructions for Windows 11 look different from the ones that seemed repeated in a different page... can you check?

File: products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md (hunk 1 above)
Comment (marciocloudflare) [reply products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md]: Thanks! I'm using Virtual Box :D

File: products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md (hunk 7 above)
Comment (marciocloudflare) [reply products/1.1.1.1/src/content/1.1.1.1-for-families/windows.md:49]: Wait.... It's not the same. Arghhh. I copy/pasted from the wrong place. Gonna change

File: products/1.1.1.1/src/content/setup-1.1.1.1/windows.md (hunk 11 above)
Comment (marciocloudflare) [reply products/1.1.1.1/src/content/setup-1.1.1.1/windows.md:36]: Yup

Comment (marciocloudflare) [reply products/1.1.1.1/src/content/setup-1.1.1.1/windows.md:36]: These are the ones that are correct. Already corrected the ones in 1.1.1.1 For Families

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**